build/FrameworkPackage/FrameworkPackageContents/AppxManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:build="http://schemas.microsoft.com/developer/appx/2015/build" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" IgnorableNamespaces="uap build mp">
+  <Identity Name="$(PackageName)" ProcessorArchitecture="$(Platform)" Publisher="$(Publisher)" Version="$(Version)" />
+  <Properties>
+    <Framework>true</Framework>
+    <DisplayName>$(PackageName)</DisplayName>
+    <PublisherDisplayName>Microsoft Platform Extensions</PublisherDisplayName>
+    <Description>Microsoft Project Reunion</Description>
+    <Logo>logo.png</Logo>
+  </Properties>
+  <Resources>
+    <Resource Language="en-US" />
+  </Resources>
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.15063.0" MaxVersionTested="10.0.18362.0" />
```

Comment:
Version | Codename | Marketing name | Build | Release date
-- | -- | -- | -- | --
1703 | Redstone 2 | Creators Update | **15063** | April 5, 2017
1709 | Redstone 3 | Fall Creators Update | 16299 | October 17, 2017
1803 | Redstone 4 | April 2018 Update | 17134 | April 30, 2018
1809 | Redstone 5 | October 2018 Update | 17763 | November 13, 2018
1903 | 19H1 | May 2019 Update | **18362** | May 21, 2019
1909 | 19H2 | November 2019 Update | 18363 | November 12, 2019
2004 | 20H1 | May 2020 Update | 19041 | May 27, 2020

https://en.wikipedia.org/wiki/Windows_10_version_history

MinVersion 10.0.15063.0 (RS2). Should be 10.0.17763.0 (RS5) ?

Likewise MaxVersion=10.0.18362.0 (19H1). Should be 10.0.19041.0 (20H1) i.e. the latest release Windows? Or should it be newer (more recent flights?)

Reply:
fixing min version, max version question was asked above.

Reply:
leaving max tested version as-is for now, will bump this in a subsequent PR.